l10n_hr_base_location/README.rst

```diff
@@ -0,0 +1,34 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Hrvatska : Gradovi i općine sa poštanskim brojevima
+===================================================
+
+Popis gradova i Županija u RH, sa pripadnim poštanskim brojevima,
+
+Izvor:
+
+Croatian counties (without HR- prefix in code)    https://en.wikipedia.org/wiki/ISO_3166-2:HR
+Hr Pošta webstranice
```

Comment:
It is relevant for the description to be also in english. Could you add that please?

Reply:
No problem, added these files, as i saw in spain localization, lots of readme files was only in spanish, so i thought for localizations descriptions are allowed to be only in local language,
from now on i will try to write all the explanations and readmes in english.

Reply:
In localization repos, README can be in native language.